course2021.html

```diff
@@ -657,7 +657,7 @@ <h3 id="seminarPresentations">Seminar presentations and posts made by students</
       <a href="">post</a>-->
                     </td>
                     <td>Team Es3-09
-                        <!--<a href="slides/course2021/seminars/SemEs3-09.pdf">slides</a>
+                        <a href="slides/course2021/seminars/SemEs3-09.pdf">slides</a>
        <a href="">post</a>-->
```

Comment:
No tenéis que añadir un enlace al documento también?

Reply:
Si si, pensaba que ya estaba puesto de serie.
También teniamos dudas de como subir la exposición por no dejarnos hacer un post directamente. Mi compañero ya le envió un mensaje al correo con este problema.

Reply:
Tenéis que hacer un pull request incluyendo el pdf de las trasparencias y el del documento...podéis hacer un commit sobre este mismo commit para actualizar la pull request